data/eos-branch

```diff
@@ -0,0 +1,37 @@
+# Endless product-specific branches configuration
+#
+# Copyright (C) 2016 Endless Mobile Inc.
```

Comment:
I don't think this necessarily needs the copyright - it's just a keyfile. But if it does, it should probably also mention the license.